frontend/src/lib/components/leaderboard.svelte

```diff
@@ -19,15 +20,40 @@
         </Table.Row>
       </Table.Header>
       <Table.Body>
-        {#each data as { loses, name, wins, mmr }, idx}
+        {#each data as { loses, name, wins, mmr, mostRecentMMRChange, winningStreak, losingStreak }, idx}
           <Table.Row>
             <Table.Cell class="w-16 font-bold">{idx + 1}</Table.Cell>
             <Table.Cell>{name}</Table.Cell>
-            <Table.Cell>{wins}</Table.Cell>
-            <Table.Cell>{loses}</Table.Cell>
-            <Table.Cell class="text-right"
-              >{mmr != 0 ? mmr : '🐣'}</Table.Cell
-            >
+            <Table.Cell>
+              {wins}
+              {#if winningStreak && winningStreak >= SHOW_STREAK_THRESHOLD}
+                <span class="ml-2 text-xs" title="Winning streak"
+                  >🔥 {winningStreak}</span
+                >
+              {/if}
+            </Table.Cell>
+            <Table.Cell
+              >{loses}
+              {#if losingStreak && losingStreak >= SHOW_STREAK_THRESHOLD}
+                <span class="ml-2 text-xs" title="Losing streak"
+                  >🌧️ {losingStreak}</span
+                >
+              {/if}
+            </Table.Cell>
+            <Table.Cell class="text-right">
+              {mmr != 0 ? mmr : '🐣'}
+              {#if mostRecentMMRChange != null && mmr != 0}
+                <span
+                  title="MMR change at last match"
+                  class={'text-xs ' +
+                    (mostRecentMMRChange > 0
+                      ? 'text-green-500'
+                      : 'text-red-500')}
+                >
+                  ({mostRecentMMRChange > 0 ? '+' : ''}{mostRecentMMRChange})
```

Comment:
I'm worried this'll make the leaderboard very busy.

Reply:
Agree, I have removed the mmr diff from this pr